package.json

```diff
@@ -28,29 +28,43 @@
   ],
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js --timeout 600000",
-    "test": "c8 mocha",
-    "fix": "eslint '**/*.js' --fix",
+    "lint": "gts fix && eslint --fix samples/*.js",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "test": "c8 mocha build/test",
+    "fix": "gts fix",
     "docs-test": "linkinator docs",
-    "predocs-test": "npm run docs"
+    "predocs-test": "npm run docs",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "^1.9.0"
   },
   "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^12.0.0",
     "c8": "^7.0.0",
     "codecov": "^3.0.0",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.0.0",
+    "gts": "^1.0.0",
     "jsdoc": "^3.5.5",
     "jsdoc-fresh": "^1.0.1",
     "jsdoc-region-tag": "^1.0.2",
     "linkinator": "^2.0.0",
-    "mocha": "^7.0.0",
-    "prettier": "^1.7.4"
+    "power-assert": "^1.6.0",
+    "prettier": "^1.13.7",
+    "mocha": "^6.1.4",
```

Comment:
Mocha 7.x is already out.  Could I trouble you to file an upstream issue on the generator to figure out a way to keep dependencies up to date?  Since we have the `package.json.njk`, renovate isn't doing anything to the file.  It's worth exploring how we do that 🤷‍♂   @alexander-fenster FYI as well

Reply:
@JustinBeckwith We exclude `package.json` from `synthtool` so renovate will fix all the dependencies very soon! Having that said - sure we'll update `mocha` to v7 in the generator.

Reply:
Huh - I wonder how mocha got downgraded here then!?

Reply:
I guess it's just the result of manual editing. When we convert an existing library from JS to TS, we write `package.json` manually, extensively using copy+paste :)

@summer-ji-eng can you revert `mocha` back to 7.x? :) Thank you!

Reply:
It might because the example PR in the conversion doc uses this, sorry for misunderstanding, I guess we may want to keep the modules up to date. That is to say, if there's duplicate modules, please pick the latest version, thanks! @summer-ji-eng

Reply:
upgrade `mocha` to the latest version.